.github/scripts/check-compose-volumes.py

```
@@ -0,0 +1,71 @@
+            return True
+        
+        # Check each top-level volume key for numeric suffix pattern
+        for vol_name in volumes.keys():
+            if isinstance(vol_name, str) and re.search(r'-[0-9]+$', vol_name):
+                print(f'ERROR: {file_path}:0: temporary volume name detected: {vol_name}')
+                print('       Volume names must not end with a numeric PR suffix (e.g. db-data-5079).')
+                print('       Rename it back to its canonical name (e.g. db-data).')
+                return False
+        
+        return True
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)